src/core/reopt_inputs.jl

```diff
@@ -974,26 +981,43 @@ function setup_ghp_inputs(s::AbstractScenario, time_steps, time_steps_without_gr
     cooling_thermal_load_reduction_with_ghp_kw = zeros(num, length(time_steps))
     ghp_cooling_thermal_load_served_kw = zeros(num, length(time_steps))        
     ghp_electric_consumption_kw = zeros(num, length(time_steps))
+    avoided_capex_by_ghp_present_value = Vector{Float64}(undef, num)
+    ghx_useful_life_years = Vector{Float64}(undef, num)
+    ghx_residual_value = Vector{Float64}(undef, num)
     if num > 0
         require_ghp_purchase = s.ghp_option_list[1].require_ghp_purchase  # This does not change with the number of options
+       
         for (i, option) in enumerate(s.ghp_option_list)
-            ghp_cap_cost_slope, ghp_cap_cost_x, ghp_cap_cost_yint, ghp_n_segments = cost_curve(option, s.financial)
-            ghp_size_ton = option.heatpump_capacity_ton
-            seg = 0
-            if ghp_size_ton <= ghp_cap_cost_x[1]
-                seg = 1
-            elseif ghp_size_ton > ghp_cap_cost_x[end]
-                seg = ghp_n_segments
-            else
-                for n in 2:(ghp_n_segments+1)
-                    if (ghp_size_ton > ghp_cap_cost_x[n-1]) && (ghp_size_ton <= ghp_cap_cost_x[n])
-                        seg = n
-                        break
-                    end
-                end
+            if option.heat_pump_configuration == "WSHP"
+                fixed_cost, variable_cost = get_ghp_installed_cost(option, s.financial, option.heatpump_capacity_ton)
+                ghp_installed_cost[i] = fixed_cost + variable_cost
+
+            elseif option.heat_pump_configuration == "WWHP"
+                temp = option.installed_cost_per_kw
+                option.installed_cost_per_kw = option.wwhp_heating_pump_installed_cost_curve
+                fixed_cost_heating, variable_cost_heating = get_ghp_installed_cost(option, s.financial, option.wwhp_heating_pump_capacity_ton)
+                ghp_installed_cost_heating = 0.5 * fixed_cost_heating + variable_cost_heating
```

Comment:
What is the 0.5 from?

Reply:
This is just to avoid double counting the "fixed cost", since GHX/aux unit costs are the same in the cost curve for both the heating heat pump and cooling heat pump. The current workaround to make incentives work mean these values aren't used since there's no y-intercept.